test/asm/multi-line-strings.asm

```diff
@@ -0,0 +1,24 @@
+S EQUS "Hello"
+
+PRINTT "\"\"\"\n"
+
+PRINTT """{S}
+world
+"""
+
+PRINTT """The multi-line string \ ; line continuations work
```

Comment:
Now that I'm thinking about it... what if an interpolated symbol ends with a backslash? I think the backslash should not be specially handled, and just be copied.

Reply:
Current `master` does not specially handle that case, so the interpolated backslash does start a line continuation.

```
S EQUS "2 + 2 \\"
	PRINTV {S}
		+ 3
	PRINTT "\n"
```

This prints `$7`.

Reply:
I meant, within a string, like here.

Reply:
Good!

Reply:
I thought `readString` sets `disableMacroArgs` and `disableInterpolation` specifically so it can do its own handling of interpolated and macro-expanded content, and include their contents directly without potential side effects like that. So in `S EQUS "{X}"`, `S` will equal the contents of `X`, even if `X` contains a quote mark, or a backslash followed by a newline.